src/edu/washington/escience/myria/operator/DbCreateUDF.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.operator;
+
+import com.google.common.collect.ImmutableMap;
+
+import edu.washington.escience.myria.DbException;
+import edu.washington.escience.myria.MyriaConstants;
+import edu.washington.escience.myria.accessmethod.AccessMethod;
+import edu.washington.escience.myria.accessmethod.ConnectionInfo;
+import edu.washington.escience.myria.storage.TupleBatch;
+
+/**
+* 
+*/
+public class DbCreateUDF extends RootOperator {
+  /** Required for Java serialization. */
+  private static final long serialVersionUID = 1L;
+
+  /** The connection to the database database. */
+  private AccessMethod accessMethod;
+  /** The information for the database connection. */
+  private ConnectionInfo connectionInfo;
+
+  private final String udfDefinition;
+
+  /**
+   * @param child the source of tuples to be inserted.
+   * @param relationKey the key of the table the tuples should be inserted into.
+   * @param connectionInfo the parameters of the database connection.
+   */
+  public DbCreateUDF(final Operator child, final String sqlString, final ConnectionInfo connectionInfo) {
+    super(child);
+    this.connectionInfo = connectionInfo;
+    udfDefinition = sqlString;
+  }
+
+  @Override
+  protected void init(final ImmutableMap<String, Object> execEnvVars) throws DbException {
+    /* Retrieve connection information from the environment variables, if not already set */
+    if (connectionInfo == null && execEnvVars != null) {
+      connectionInfo = (ConnectionInfo) execEnvVars.get(MyriaConstants.EXEC_ENV_VAR_DATABASE_CONN_INFO);
+    }
+    /* Open the database connection */
+    accessMethod = AccessMethod.of(connectionInfo.getDbms(), connectionInfo, false);
+
+    /* Drop the table */
+    accessMethod.runCommand(udfDefinition);
```

Comment:
What is the purpose of defining a separate operator for UDF creation when it simply executes an arbitrary user-provided SQL statement (with no validation!)? Why not just have the `createUDF` API call a generic `DbExecute` operator?